src/bun.js/bindings/Strong.cpp

```diff
@@ -1,37 +1,58 @@
 #include "root.h"
+#include <JavaScriptCore/Strong.h>
 #include <JavaScriptCore/StrongInlines.h>
 #include "BunClientData.h"
 #include "wtf/DebugHeap.h"
-#include "Strong.h"
-namespace Bun {
+#include "ZigGlobalObject.h"
 
-#if ENABLE(MALLOC_BREAKDOWN)
-DEFINE_ALLOCATOR_WITH_HEAP_IDENTIFIER(StrongRef);
-#endif
-
-}
-
-extern "C" void Bun__StrongRef__delete(Bun::StrongRef* strongRef)
+extern "C" void Bun__StrongRef__delete(JSC::JSValue* handleSlot)
 {
-    delete strongRef;
+    if (handleSlot) {
+        // deallocate() will correctly remove the handle from the strong list if it's currently on it.
+        JSC::HandleSet::heapFor(handleSlot)->deallocate(handleSlot);
+    }
```

Comment:
```suggestion
    // deallocate() will correctly remove the handle from the strong list if it's currently on it.
    JSC::HandleSet::heapFor(handleSlot)->deallocate(handleSlot);
```

Looks like this would be a bug if it was null

Reply:
Same with the other Bun__StrongRef__ functions. I think instead they should `ASSERT(handleSlot)`